lib/secure_headers.rb

```diff
@@ -30,14 +39,60 @@ module SecureHeaders
   module Configuration
     class << self
```

Comment:
Yeah, that will also make it easier to scope the configuration for the given precomputed overrides.